dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -25,7 +25,7 @@ function create_partitions {
     if type partprobe &> /dev/null;then
         partprobe "${disk_device}"
     else
-        blockdev --rereadpt "${disk_device}"
+        partx -u "${disk_device}"
```

Comment:
@schaefi I think partx should be added to module-setup.sh for both modules. For the live module it is enough to swap it for blockdev (since it is no longer used there). For kiwi-lib then both are required `blockdev` and `partx`. Both tools a provided by util-linux, hence no spec changes are required.

Reply:
yes, fixed